app/scripts/constants/event-names.js

```diff
@@ -0,0 +1,8 @@
+export const EVENTS = {
+  // Fired after the actively selected network is changed
+  NETWORK_DID_CHANGE: 'networkDidChange',
+  // Fired when the actively selected network *will* change
+  NETWORK_WILL_CHANGE: 'networkWillChange',
+  // Fired after state changes that impact the extension badge (unapproved msg count)
+  UPDATE_BADGE: 'updateBadge',
```

Comment:
I'm not sure it makes sense to group these together :thinking: The first two are network controller events, but the third is a MetaMask controller event. They're emitted on entirely different things.

Or at the very least this difference should be more prominent in the comment descriptions.

Reply:
@Gudahtt do you have any preference for one of the following:

a. Controller specific event constants declared and exported in the controller file as `EVENTS` with events that are emitted by many controllers in the `metamask-controller` file
b. Group events in this `event-names` file by controller name. `NETWORK.WILL_CHANGE` and `METAMASK_CONTROLLER.UPDATE_BADGE` 

Either option is good for me, and I agree with your feedback.

Reply:
Either one seems fine! These will likely be supplanted by the new controller messaging system in the future anyway I would imagine.